python_modules/dagster/dagster/core/host_representation/external_data.py

```diff
@@ -336,6 +336,14 @@ def __new__(cls, error):
         )
 
 
+@whitelist_for_serdes
+class ExternalNotebookData(namedtuple("_ExternalNotebookData", "content")):
+    def __new__(cls, content):
+        return super(ExternalNotebookData, cls).__new__(
+            cls, content=check.str_param(content, "content")
+        )
```

Comment:
for file data seems like we should be doing bytes or something not this json obj + string